src/test/groovy/NotificationManagerStepTests.groovy

```diff
@@ -450,6 +450,145 @@ class NotificationManagerStepTests extends ApmBasePipelineTest {
     assertJobStatusSuccess()
   }
 
+  @Test
+  void test_notify_slack_with_multiple_channels() throws Exception {
+    def script = loadScript(scriptName)
+    try {
+      script.notifySlack(
+        build: readJSON(file: "build-info-manual.json"),
+        buildStatus: "SUCCESS",
+        changeSet: readJSON(file: "changeSet-info-manual.json"),
+        stepsErrors: readJSON(file: "steps-errors.json"),
+        testsErrors: readJSON(file: "tests-errors.json"),
+        testsSummary: readJSON(file: "tests-summary.json"),
+        channel: 'foo, bar, baaz',
+        credentialId: 'test',
+        enabled: true
+      )
+    }
+    catch(e) {
+      println e
+    }
+    printCallStack()
+    assertFalse(assertMethodCallContainsPattern('log', 'notifySlack: Error with the slack comment'))
+    assertTrue(assertMethodCallContainsPattern('slackSend', 'channel=foo'))
+    assertTrue(assertMethodCallContainsPattern('slackSend', 'channel=bar'))
+    assertTrue(assertMethodCallContainsPattern('slackSend', 'channel=baaz'))
```

Comment:
@v1v I finally avoided using `log` but checking that `slackSend` was invoked.

The `channel=baaz` value has been checked against a debug execution of the tests, where the output of markdown template and slack send step is printed.